src/components/WelcomeModal/WelcomeModal.tsx

```diff
@@ -75,112 +75,66 @@ const WelcomeModal = () => {
     } catch (e) {}
   };
   return (
-    <Modal
-      centered
-      footer={
-        <Button
-          variant={Variant.Primary}
-          onClick={handleWelcomeClosed}
-          data-cy="close-welcome-modal"
+    <ClassNames>
+      {({ css }) => (
+        <Modal
+          setOpen={handleClosed}
+          open={visible}
+          data-cy="welcome-modal"
+          className={css`
+            z-index: 1000;
+          `}
         >
-          Done
-        </Button>
-      }
-      onCancel={handleWelcomeClosed}
-      style={{ top: 20 }}
-      visible={visible}
-      width={800}
-      maskStyle={{
-        backgroundColor: black,
-        opacity: 0.9,
-      }}
-      wrapProps={{
-        "data-cy": "welcome-modal",
-      }}
-    >
-      {/* @ts-expect-error */}
-      <CardTitle>Welcome to the New Evergreen UI!</CardTitle>
-      <Carousel
-        afterChange={(number) => setActiveSlide(number)}
-        autoplay
-        autoplaySpeed={8000}
-        dots={false}
-        slickGoTo={activeSlide}
-        ref={slider}
-      >
-        {carouselCards.map((card) => (
-          <CarouselCard key={`card_${card.img}`} {...card} />
-        ))}
-      </Carousel>
-      <CarouselDots
-        activeSlide={activeSlide}
-        cards={carouselCards}
-        slider={slider}
-      />
-    </Modal>
+          {/* @ts-expect-error */}
+          {title && <CardTitle>{title}</CardTitle>}
+          <Carousel
```

Comment:
This carousel is pretty off-center in the modal

<img width="651" alt="image" src="https://user-images.githubusercontent.com/9298431/166980543-206a5188-1283-49f4-bf4d-8805e2a56db5.png">

Reply:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/4605522/167226261-58d762b1-e2b1-49ab-95fc-b3c0986e1327.png">

The designs actually have them as being left aligned. I ended up trimming the images and videos and it helped with the positioning.